_episodes/interviews/2022-12-23-matt-butcher.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Fermyon with Matt Butcher"
+date: 2022-12-23T16:51:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-12-23-matt-butcher.mp3
+duration: "54:19"
+length: "52142916"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Matt Butcher](https://twitter.com/technosophos), CEO of [Fermyon](https://www.fermyon.com/), who is working on [Fermyon Cloud](https://www.fermyon.com/cloud), a platform to effortlessly deploy and host your WebAssembly applications.
```

Comment:
```suggestion
Allen Wyma talks with [Matt Butcher](https://twitter.com/technosophos), CEO of [Fermyon](https://www.fermyon.com/), who is working on [Fermyon Cloud](https://www.fermyon.com/cloud), a platform to deploy and host WebAssembly applications.
```
Let's not do a sales pitch in the notes :)

Also, what's the connection to Rust here?

Reply:
@jonhoo let me know if this suffices:

Fermyon is nearly all written in Rust and WebAssembly has first class Rust support. Not to mention, Matt has extensive use of Rust with his involvement in Krustlet and various other projects written in Rust as we discuss within the episode.

Reply:
@allenwyma Yes, that's great thanks!